distribution/std-lib/Database/src/Data/Table.enso

```diff
@@ -43,14 +47,16 @@ type Table
 
        Converts this table to a JSON structure.
     to_json : Json
-    to_json = this.to_sql.to_json
+    to_json = case this.internal_columns.is_empty of
+        True -> "Table: no columns".to_json
```

Comment:
This is a very bad json representation. Use an object with fields or something like that, not "here's a random string"

Reply:
```suggestion
        True -> Json.from_pairs [["query", Nothing], ["message", "The table has no columns so a query cannot be generated."]]
```

Does this make sense?